projects/wave-core-new/src/lib/layers/legend/legend.component.ts

```diff
@@ -9,6 +7,4 @@ import {AbstractSymbology} from '../symbology/symbology.model';
     selector: 'wave-legendary',
     templateUrl: 'legend.component.html',
 })
-export class LegendComponent<S extends AbstractSymbology> {
-    @Input() symbology: S;
-}
+export class LegendComponent {}
```

Comment:
What is this component for?

Reply:
that's what I am wondering... should I just remove it and rename the MappingRasterLegend to it?

Reply:
I guess you now know which is used and which is not needed anymore, so yeah, remove duplicates.

Reply:
it is actually used in the data atlas so I didn't remove it